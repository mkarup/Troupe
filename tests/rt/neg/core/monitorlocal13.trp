let val p = spawn (fn () => sleep 750)
in
  monitorlocal (p, 42)
end