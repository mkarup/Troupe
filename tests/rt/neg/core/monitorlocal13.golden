2024-12-11T16:27:06.562Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread 6b48284b-dcaf-48ea-a7f3-51a6480613de@{}%{}
>> value 42@{}%{} is not a level
